cmd/ovirt-populator/Makefile

```diff
@@ -0,0 +1,8 @@
+GOBIN ?= ${GOPATH}/bin
+CONTROLLER_GEN=$(GOBIN)/controller-gen
+
+manifests:
+	${CONTROLLER_GEN} crd rbac:roleName=manager-role webhook paths="./pkg/..." output:crd:artifacts:config=config/crds/bases output:crd:dir=config/crds
+
+generate:
```

Comment:
all but `main.go` should move out of `cmd` - `ovirtpopulator.go` should be placed under `pkg/apis/..` and then it would be picked up by the existing targets in the main Makefile